src/main/kotlin/com/faforever/userservice/domain/UserService.kt

```diff
@@ -62,7 +62,7 @@ class UserService(
     fun findUserBySubject(subject: String) =
         userRepository.findById(subject.toLong())
 
-    private fun checkLoginThrottlingRequired(ip: String) = loginLogRepository.findFailedAttemptsByIp(ip)
+    private fun checkLoginThrottlingRequired(ip: String) = loginLogRepository.findFailedAttemptsByIpAfterDate(ip, LocalDateTime.now().minusDays(1))
```

Comment:
number of days should be configurable in properties with a default value, as all other security related thresholds do

Reply:
ah yes good point